### PR TITLE
Feature/extd 51 download all possible formats connect with style context menu

### PR DESCRIPTION
**Pull request by awaitDzn**

### CODE REVIEW

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/components/download-image/download-image.component.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+} from '@picturepark/sdk-v1-angular';
+import { map } from 'rxjs/operators';
+import { Subscription } from 'rxjs';
+
+@Component({
+  selector: 'pp-download-image',
+  templateUrl: './download-image.component.html',
+  styleUrls: ['./download-image.component.scss']
+})
+export class DownloadImageComponent implements OnInit, OnDestroy {
```
Comment (neotek) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/components/download-image/download-image.component.ts]: @awaitDzn Rename to OutputDownloadMenuComponent

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/components/download-image/download-image.component.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  @Input() public id: string;
+  @Input() public formats: string[];
+  @Input() public overlapTrigger: boolean = false;
+  // possible values  'above' | 'below'
+  @Input() public yPosition: string = 'above';
+  // possible values 'before' | 'after'
+  @Input() public xPosition: string = 'before';
+
+  public outPutFormats: string[];
```
Comment (neotek) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/components/download-image/download-image.component.ts]: rename to outputFormats

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/components/download-image/download-image.component.ts (hunk 1 above)
Comment (awaitDzn) [reply src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/components/download-image/download-image.component.ts]: @neotek done

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/components/download-image/download-image.component.ts (hunk 2 above)
Comment (awaitDzn) [reply src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/components/download-image/download-image.component.ts]: @neotek done

**Pull request merged**